client/src/featureform/register.py

```diff
@@ -3835,7 +3841,12 @@ def register_entity(
 
     def register_column_resources(
         self,
-        source: Union[NameVariant, SourceRegistrar, SQLTransformationDecorator],
+        source: Union[
+            NameVariant,
+            SourceRegistrar,
+            SQLTransformationDecorator,
```

Comment:
I plan on making a new class for these two decorators called SourceVariantProvider